### ISSUE TITLE
First release result collection

### ISSUE THREAD

**Issue by tikikun** [homebrewltd/llama3-s#30]
Check list:
- Training result
- MMLU for intermediate results to check for degradation
- Saving checkpoints to internal cloud storage
- Training specs

**Comment by bachvudinh** [comment]
- MMLU for intermediate results please check here: https://www.notion.so/jan-ai/Llama3-s-Worklog-d99aa14d563942fabb5b003df48f7d41.
- Checkpoints are uploaded to S3 with Bucket name: research19871
- Training specs will be uploaded along with the model checkpoint after the training finish 
cc @tikikun

**Comment by bachvudinh** [comment]
Here are some results of our model at first epoch:
![Screenshot from 2024-08-19 17-01-42](https://github.com/user-attachments/assets/50820852-7081-4618-888a-c0554777f962)
![Screenshot from 2024-08-19 17-01-15](https://github.com/user-attachments/assets/cedbb519-cab3-4ba1-8427-2833037a44ce)
![Screenshot from 2024-08-19 17-00-08](https://github.com/user-attachments/assets/b2cd143f-64f5-41db-8265-43f4cdaec340)

In conclusion, Our model is very good at understanding sementic audio
